pkg/users/account-create-dialog.js

```diff
@@ -50,6 +50,18 @@ function AccountCreateBody({ state, errors, change, shells }) {
         shell, isShellSelectExpanded,
     } = state;
 
+    // We want to let user know that password and confirmation password do not match without them having to constantly click on "Create" to validate the form.
+    // But we also don't want to show an error message while they are typing the confirmation password, only when they are finished typing it.
+    // To solve the issue of telling that user is finished writing the confirmation password, let's do the following:
+    // The dialog does not validate if passwords match until:
+    //     1. they are at least the same length (which signals user has finished typing)
+    // OR
+    //     2. user submits the form (which also signals they are finished typing)
+    // Once that happens, and passwords do not match, the confirm password is validated after each keystroke.
+    let dynamic_password_confirm_error;
+    if (state.password_confirm_dirty)
+        dynamic_password_confirm_error = validate_password_confirm(state.password_confirm, state.password);
```

Comment:
Code style: Whenever possible, use `const`. Here this is quite straightforward:

    const dynamic_password_confirm_error = state.password_confirm_dirty && validate_password_confirm(..);

(with a line break)